File: benchmarks/static/corpus/go/vuln.go
package p
import ("os/exec"; "os"; "crypto/md5"; "database/sql")
func h(u, up string, db *sql.DB) {
    exec.Command("sh", "-c", u)   // EXPECT BHF-404
    os.ReadFile(up)               // EXPECT BHF-405
    md5.New()                     // EXPECT BHF-422
    db.Query("SELECT * FROM t WHERE x=" + u)  // EXPECT BHF-419
}

File: benchmarks/static/corpus/java/Vuln.java
class Vuln { void h(String u, java.io.ObjectInputStream ois, java.sql.Statement st) throws Exception {
    Runtime.getRuntime().exec("sh -c " + u);              // EXPECT BHF-404
    st.executeQuery("SELECT * FROM t WHERE x=" + u);      // EXPECT BHF-419
    Object o = ois.readObject();                          // EXPECT BHF-421
    java.security.MessageDigest.getInstance("MD5");       // EXPECT BHF-422
    String apiKey = "AKIAsecretvalue1234";                // EXPECT BHF-429
} }
